### ISSUE TITLE
Add support for design/graphic unit conversion

### ISSUE THREAD

**Issue by xavierfoucrier** [veler/DevToys#943]
### What improvement do you think would an existing feature or tool in DevToys?

It would be nice to add design/graphic unit conversion, like `px` to `rem`, `inch`, `em`, `cm`, etc.

### Solution/Idea

Add a dedicated sub section called "Unit's" inside the "Graphic" section.

**Comment by Regenhardt** [comment]
Doesn't px:cm depend on a screen's individual DPI? How would we calculate that? Use current screen? Ask for DPI?

**Comment by veler** [comment]
> Doesn't px:cm depend on a screen's individual DPI? How would we calculate that? Use current screen? Ask for DPI?

I'm wondering if this could be detected automatically...

**Comment by heribertolugo** [comment]
> Doesn't px:cm depend on a screen's individual DPI? How would we calculate that? Use current screen? Ask for DPI?

I believe you can do px:cm|in, fairly accurately. However, rem completely depends on what the root font & font size are in a web page and browser font settings (possibly font availability in the system). And em also, except em would change from container to container - so it could mean different values in the same page. 

![Screenshot_20231101_221606_Chrome.jpg](https://github.com/veler/DevToys/assets/26213368/3f968572-3102-4770-9a5f-828bafe79ac8)

To convert to those other units a lot of variables/ factors would need to be considered- and I don't think any of the conversions can guarantee 100% accuracy. 

Regardless, it might still be handy for web developers.

**Comment by xavierfoucrier** [comment]
For px/rem units, you only need the base font size which could be defined to 16px by default.
> See https://pixelsconverter.com/px-to-rem.

**Comment by heribertolugo** [comment]
yup for rem. but em (and i believe ex too) rely on parent container, which then rely on parent container, and so on.

**Comment by xavierfoucrier** [comment]
@heribertolugo I agree for `em` units.

**Comment by heribertolugo** [comment]
not sure what you mean. are we agreeing on the similarity between em vs rem, and the difference is how one's value propagates through ancestor elements and the other doesnt? lol

**Comment by xavierfoucrier** [comment]
Hello everyone, any news on this?
Basically the idea here is to provide a section for web developers that work with `px` and `rem` everyday.
Same as https://pixelsconverter.com/px-to-rem.

Let me know, if not I will close the issue 😉
